kerboscript_tests/functest30.ks

```diff
@@ -0,0 +1,8 @@
+print "Testing the case of a RUN in a loop.".
```

Comment:
I added this test case because I was messing with the area of the code where terminal RUN differs from in-program RUN, and I wanted to make sure that when doing a re-run of the same subprogram, where it skips the compile step and just uses the previously compiled version, that I hadn't broken that case.  I noticed that none of the existing functests covered this case properly.